src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -90,10 +118,26 @@ public int countImages() {
     return images.size();
   }
 
+  public int countHeaders() {
+    return headers.size();
```

Comment:
this can be unstable, you should wait for it's item to be loaded or wait for element after this, so you will be sure that all elements are rendered